examples/gallery/lines/contours.py

```diff
@@ -0,0 +1,41 @@
+"""
+Contours
+--------
+The :meth:`pygmt.Figure.contour` method can plot contour lines from a table of points by direct triangulation.
+The data to the triangulation can by provided in one of three options:
+    1: ``x``, ``y``, ``z`` 1d :class:`numpy.ndarray` data columns\n
+    2: ``data`` 2d :class:`numpy.ndarray` data matrix with 3 columns corresponding to ``x``, ``y``, ``z``\n
+    3: ``data`` path string to a file containing the ``x``, ``y``, ``z`` in a tabular format\n
+The parameters ``levels`` and ``annotation`` are deciding on the contours intervals and intervals of the
+annotation on the contours respectively.\n
```

Comment:
```suggestion
annotation on the contours respectively.

```
In sphinx with ReStructuredText (rst), a blank line is used to make sure that the next sentence is displayed on a new line.